Competition2017/src/competition/subsystems/autonomous/selection/AutonomousCommandSelector.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.autonomous.selection;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.command.Command;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.properties.StringProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class AutonomousCommandSelector extends BaseSubsystem {
```

Comment:
This is class is going to be used by a seperate command class which using this class sets the autonomous command